MASH-FRET/source/project/setParam.m

```diff
@@ -360,6 +368,8 @@
 p.proj = {};
 p.curr_proj = 0;
 p.curr_mol = [];
+p.defTagNames = {'static', 'dynamic'};
```

Comment:
Should we add a few other default ones to help the user, e.g. D-only, A-only?

Reply:
Thank you, A-only and D-only are now added in MASH-code-6